master/static/srv/notebook-entrypoint.sh

```diff
@@ -18,6 +18,30 @@ set -x
 test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 set +x
 
+if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ]; then
+    version=$(echo $(curl -s -X POST -H "Content-Type: application/json" pachyderm-proxy:80/api/versionpb_v2.API/GetVersion) | sed -n 's/.*"major":\([0-9]*\), "minor":\([0-9]*\), "micro":\([0-9]*\).*/\1.\2.\3/p')
```

Comment:
Is pachyderm_proxy a kubernetes service or something that you are assuming exists? This may not be feasible in general, right?

Reply:
why would this not use PACHD_ADDRESS if i'm right?

Reply:
Yes pachyderm-proxy is a Kubernetes service that will always be there. The pachd address can't be used because pachd doesn't expose an http port.

Reply:
Is it always guaranteed to be exactly this? What if this notebook is getting ran in a namespace / cluster that is not where pachyderm is installed? Maybe we can just setup a chat.

Reply:
The most recent commit addresses the different namespace issue. This would only be supported when using the unified install.